lgsvl_pkgs/lgsvl_msgs/CHANGELOG.rst

```diff
@@ -0,0 +1,12 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
As mentioned in https://github.com/ApolloAuto/apollo/pull/11187, we'd prefer to maintain such modules in `modules/contrib`.

Reply:
Moved.